Description: set default file chooser folder to examples
 This sets the default dir for the build profile file chooser to be
 the examples folder that is included in this package, which includes
 a few build profiles.
Author: Hans-Christoph Steiner <hans@eds.org>
Forwarded: not-needed

Index: dummydroid/src/main/java/de/onyxbits/dummydroid/LoadForm.java
===================================================================
--- dummydroid.orig/src/main/java/de/onyxbits/dummydroid/LoadForm.java
+++ dummydroid/src/main/java/de/onyxbits/dummydroid/LoadForm.java
@@ -21,7 +21,7 @@ public class LoadForm extends AbstractFo
 
 	public LoadForm(NavigateAction forwardAction, NavigateAction backwardAction) {
 		super(forwardAction, backwardAction);
-		fileChooser = new JFileChooser(new File(System.getProperty("user.dir")));
+		fileChooser = new JFileChooser(new File("/usr/share/doc/dummydroid/examples"));
 		fileChooser.setFileFilter(new BuildPropFilter());
 		properties = new Properties();
 		add(fileChooser);
